sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill.md

```diff
@@ -0,0 +1,83 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
+applicable: SharePoint Online
+title: Start-SPOSiteOpticalCharacterRecognitionBackfill
+schema: 2.0.0
+author: FarreltinF
+ms.author: fanyi
+ms.reviewer:
+---
+
+# Start-SPOSiteOpticalCharacterRecognitionBackfill
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization
+
+Initiates a job to trigger the Optical Character Recognition (OCR) process for all files for the selected site.
```

Comment:
>Optical Character Recognition (OCR) [](http://example.com/codeflow?start=31&length=35)

simply use "OCR" going forward, since you already define the acronym in previous line